lisa/tools/iperf3.py

```diff
@@ -222,10 +224,22 @@ def run_as_client_async(
             f"{self.command} {cmd}", shell=True, sudo=True
         )
 
-        # IPerf output emits lines of the following form when it is running
-        # 132.00-133.00 sec   167 MBytes  1.40 Gbits/sec    5    626 KBytes
-        # check if stdout buffers contain "bits/sec" to determine if running
-        process.wait_output("bits/sec")
+        if log_file:
+            timeout = 60
+            timer = create_timer()
+            while timeout > timer.elapsed(False):
+                cat = self.node.tools[Cat]
+                iperf_log = cat.read(log_file, sudo=True, force_run=True)
+                if "Connecting to host " in iperf_log:
```

Comment:
Can it wait for wait_output with a timeout?

Reply:
while timeout > timer.elapsed(False): is a timeout?

Reply:
Is there any output to console to check?

Reply:
when specify the log_file, all output write into the log_file

Reply:
I see.

1. Add sleep(1) to prevent it call too frequently.
2. Add argument "no_debug_log" to cat.read not to output to console to reduce useless log.

Reply:
update